src/test/java/com/puppycrawl/tools/checkstyle/internal/AllTestsTest.java

```diff
@@ -249,7 +253,8 @@ private static boolean shouldSkipInputFileNameCheck(String path, String fileName
                 // special directory for files that can't be renamed or are secondary inputs
                 || path.contains(File.separatorChar + "inputs" + File.separatorChar)
                 // all inputs must start with 'messages'
-                || path.contains(File.separatorChar + "translation" + File.separatorChar);
+                || path.contains(File.separatorChar + "translation" + File.separatorChar)
+                || ".DS_Store".equals(fileName);
```

Comment:
`".DS_Store"` what is this file?

Reply:
It is file from Mac OS

Reply:
may be it is better to exclude all files with names that start with dot?

Reply:
I agree.

Reply:
done

Reply:
@gaurabdg , please remove this line, after some thinking ..... it HACK. We should not allow users to create unexpected files in repo and treat it as ok.

Reply:
It was changed to files starting with `.`, as merged in master